src/primitives/transaction.h

```diff
@@ -302,6 +303,10 @@ class CTransaction
     const std::vector<CTxOut> vout;
     const uint32_t nLockTime;
     const std::vector<CPourTx> vpour;
+    // TODO: This should be an unsigned char[33] (or boost array)
```

Comment:
Warning! That diff is out of date due to later commits in this PR. I recommend reviewing based on full changes.